docs/admin/external-auth.md

```diff
@@ -11,22 +11,79 @@ application. The following providers are supported:
 
 The next step is to configure the Coder server to use the OAuth application by
 setting the following environment variables:
+<div class="tabs">
+
+## Environment Variables
```

Comment:
```suggestion
## Configuration
```
Thoughts on calling this section **Configuration** and the following section **Application** (i.e. to use in terraform and workspace CLI)